test/integration.js

```diff
@@ -1,11 +1,19 @@
 const test = require('ava');
 const execa = require('execa');
 
+test.before(async () => {
+	if (process.env.GITHUB_ACTIONS) {
+		console.log('running on Github actions');
+		await execa('git', ['config', '--global', 'user.name', 'Github actions']);
+		await execa('git', ['config', '--global', 'user.email', 'actions@github.com']);
```

Comment:
This too you can do in the GH Actions file and not here.

https://github.com/fregante/setup-git-user